test/worker/move-contact-worker.spec.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable dot-notation */
+import axios from 'axios';
+import { expect } from 'chai';
+import sinon, { SinonSandbox } from 'sinon';
+
+import Auth from '../../src/lib/authentication';
+import { queueManager } from '../../src/shared/queues';
+import { 
+  MoveContactWorker, 
+  MoveContactData, 
+  JobResult 
+} from '../../src/worker/move-contact-worker';
+
+describe('MoveContactWorker', () => {
```

Comment:
nit: generally the describe in this project has the path to the file rather than the name of a class. your call to follow